www/scratch-website.studio-l10njson/fr.json

```diff
@@ -19,10 +19,10 @@
     "studio.addProject": "Ajouter",
     "studio.addProjectPlaceholder": "URL du projet",
     "studio.openToAll": "Tout le monde peut ajouter des projets",
-    "studio.addProjects.noSharedYet": "You don’t have shared projects that you can add to this studio yet.",
-    "studio.addProjects.noFavoritedYet": "You don’t have favorite projects that you can add to this studio yet.",
-    "studio.addProjects.noRecentYet": "You don’t have recently viewed projects that you can add to this studio yet.",
-    "studio.addProjects.noStudentsYet": "You don’t have student projects that you can add to this studio yet.",
+    "studio.addProjects.noSharedYet": "Vous n'avez pas encore de projets partagés que vous pouvez ajouter à ce studio.",
```

Comment:
to avoid conflicts it would be better not to include the fr.json change.

Reply:
Discussed in chat:

> cwillisf
> [...] For future reference, would you say it's better in an "add a language" PR to remove (`git restore --staged` / `git checkout`) any files not related to the language being added?

> ChrisG
> Yes, avoiding any files unrelated to the language is a good idea. Since you’ve already resolved the conflicts, it’s fine to just land this as is. :slightly_smiling_face: